src/fdc3/js/composeui-fdc3/package.json

```diff
@@ -23,6 +23,14 @@
     "jest-environment-jsdom": "^29.7.0",
     "rxjs": "^7.8.1"
   },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/morganstanley/ComposeUI.git"
+  },
+  "publisConfig": {
```

Comment:
Should this not be publishConfig?

Reply:
those typos.... thanks will update the PR! :D